etl-data-pipeline/src/main/resources/application.yaml

```diff
@@ -0,0 +1,38 @@
+spring:
+  kafka:
+    producer:
+      value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
+      key-serializer: org.springframework.kafka.support.serializer.StringOrBytesSerializer
+    stream:
+      input:
```

Comment:
If we want to add new topics, does this get updated while the application is in service or it needs to be defined before every docker build?

Reply:
If you are going to use the consolidated docker images i.e., zookeeper, kafka and the etl service built by the docker compose, then yeah we have to rebuild everytime when we have add a new topic or update the code.